ts/src/binance.ts

```diff
@@ -7373,7 +7373,10 @@ export default class binance extends Exchange {
         if (!rational) {
             initialMarginPercentageString = Precise.stringAdd (initialMarginPercentageString, '1e-8');
         }
-        const initialMarginString = Precise.stringDiv (Precise.stringMul (notionalStringAbs, initialMarginPercentageString), '1', 8);
+        let initialMarginString = this.safeString (position, 'isolatedWallet');
```

Comment:
There is field `isolatedWallet` : https://binance-docs.github.io/apidocs/futures/en/#position-information-v2-user_data:~:text=%220%22%2C%2C-,%22isolatedWallet%22%3A%20%220%22,-%2C%0A%20%20%20%20%20%20%20%20%22symbol%22
that carries that value, soo we dont need calculations here

Reply:
We might consider this change, but I'm not sure how this fixes the pointed bug. That bug might appear elsewhere if the difference is in some Precise function

Reply:
no, maybe i didnt express it well. this does not fix the aforementioned issue we discussed on discord. this is just an additional bugfix (enhancement) inside position parsing specifically for binance, that i've found down the road. 
so, this PR title might have been different, sorry.

Reply:
wait, fixing some issue in this PR

Reply:
done. now ready to review